metaflow/metaflow_config_funcs.py

```diff
@@ -59,9 +59,15 @@ def from_conf(name, default=None, validate_fn=None):
     validate_fn should accept (name, value).
     If the value validates, return None, else raise an MetaflowException.
     """
-    env_name = "METAFLOW_%s" % name
     is_default = True
-    value = os.environ.get(env_name, METAFLOW_CONFIG.get(env_name, default))
+    env_name = "METAFLOW_%s" % name
+    if prefix:
+        len_prefix = len(env_name) + 1
```

Comment:
```suggestion
        len_prefix = len(env_name)
```

Since the underscore is now explicit in the prefix arg, this isn't needed. It feels easier to read to have it called out explicitly.

Reply:
I agree!